### PR TITLE
ENG-1276 CMS widget renaming

### PR DESCRIPTION
**Pull request by jeffgo10**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **pietrangelo**

If they are not already assigned, you can assign the PR to them by writing `/assign @pietrangelo` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-plugin-jacms).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-plugin-jacms/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/main/resources/sql/plugins/jacms/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/plugins/jacms/port_data_production.sql:55]: please revert this

File: src/main/resources/sql/plugins/jacms/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/plugins/jacms/port_data_production.sql:75]: please revert this

**Review by gidesan — CHANGES_REQUESTED** [review]
Hey Jeff, just double checked with PM. Only the Search Result --> Search Results is really needed. Sorry for the misunderstanding.

**Review by gidesan — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-plugin-jacms/pull/43#pullrequestreview-512472717" title="Approved">gidesan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-plugin-jacms).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/entando-plugin-jacms/blob/master/OWNERS)~~ [gidesan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
